packages/core/admin/admin/tests/server.ts

```diff
@@ -1,10 +1,10 @@
 import { rest } from 'msw';
-import { setupServer } from 'msw/node';
+import { SetupServer, setupServer } from 'msw/node';
 import * as qs from 'qs';
 
 import { MockData, mockData } from './mockData';
 
-export const server = setupServer(
+export const server: SetupServer = setupServer(
```

Comment:
What's the return type of setupServer? I'm surprised this is needed

Reply:
It might not be now, this might be a hangover of a few things I tried to do, whilst it's not a problem, I'll remove it for simplicity!